### ISSUE TITLE
Cannot find module - crash when trying to run whackage

### ISSUE THREAD

**Issue by kennethlynne** [FormidableLabs/whackage#14]
```
> node node_modules/react-native/local-cli/cli.js start "--config" "/Users/kennethlynne/Documents/GitHub/differ-mobile/node_modules/whackage/src/packager/rn-cli.config.js"

module.js:472
    throw err;
    ^

Error: Cannot find module '/Users/kennethlynne/Documents/GitHub/differ-mobile/node_modules/react-native/packager/blacklist'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/kennethlynne/Documents/GitHub/differ-mobile/node_modules/whackage/src/packager/rn-cli.config.js:2:19)
    at Module._compile (module.js:571:32)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/kennethlynne/Documents/GitHub/differ-mobile/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
```

Using react-native 0.48.4

**Comment by matt-oakes** [comment]
Same issue with RN 0.49.3

**Comment by admmasters** [comment]
This is because react-native has extracted metro bundler. Is this being maintained anymore? You can try using my fork: https://github.com/admmasters/whackage

**Comment by admmasters** [comment]
Think this can be closed now :) Good to know its still being maintained!

**Comment by kennethlynne** [comment]
Fixed in 1.2.0 (not released on npm yet tho)

**Comment by jevakallio** [comment]
Released in https://github.com/FormidableLabs/whackage/releases/tag/v1.2.0 now!
